### DISCUSSION TITLE
Floating Toggle "remember" position

### DISCUSSION THREAD

**Discussion by Nidkid** [herbstluftwm/herbstluftwm#1545] [Q&A]
Hello,

I'm attempting to make a script which extends the floating toggle functionality by taking it to another tag and then bringing it back. I have everything set up to do so, the only issue is that the window doesn't "remember" its previous position when I untoggle it, unlike when everything stays on the same tag. 

I know I can get the frame index though `herbstclient attr clients.focus.parent_frame' but is there any way to specify the exact location within the frame so I can set it again later?

**Comment by Nidkid** [comment]
Nvm I can do this through `tags.focus.tiling.<previous index>.selection`, then `bring`

**Reply by t-wissmann** [reply]
There is no way to find the index of a non-focused window in a frame, but I'm glad that you found a workaround!
